scrapyd_k8s/api.py

```diff
@@ -1,17 +1,22 @@
 #!/usr/bin/env python3
+import logging
+from .config import Config
+config = Config()
+from .logging_config import setup_logging
+logging_level = config.scrapyd().get('logging_level', 'INFO')
```

Comment:
Can we use `log_level` instead? I have seen that much more often in various projects.